test/evergreen.yml

```diff
@@ -68,8 +68,10 @@ functions:
         set -o errexit
         set -o verbose
         if [ "Windows_NT" == "$OS" ]; then
-          pip install scons==3.1.1
-          scons-3.1.1.bat ${win_configure_flags|--enable-python=c:\\swigwin-3.0.2\\swig.exe --enable-diagnostic} ${smp_command|}
+          export "PATH=/cygdrive/c/Python39:/cygdrive/c/Python39/Scripts:$PATH"
+
+          python -m pip install scons==3.1.1
+          scons-3.1.1.bat ${win_configure_flags|"LIBPATH=c:\\python\\Python37\\libs" --enable-python=c:\\swigwin-3.0.2\\swig.exe --enable-diagnostic} ${smp_command|}
```

Comment:
`win_configure_flags` evergreen variable seems not being used elsewhere. I think it's safe to clean it up - only keep the contents after `|`.  

I believe Python39 binary is able to work with Python37 libs, it would be nice if Python39 libs can be used here if they are available on the Evergreen Windows hosts.

Reply:
Fixed - using Python39 binary and libs.